autosubmit/platforms/platform.py

```diff
@@ -18,12 +18,13 @@
 import time
 
 
+
 def recover_platform_job_logs_wrapper(platform, recovery_queue, worker_event, cleanup_event):
     platform.recovery_queue = recovery_queue
     platform.work_event = worker_event
     platform.cleanup_event = cleanup_event
     platform.recover_platform_job_logs()
-
+    _exit(0)  # Exit userspace after manually closing ssh sockets, recommended for child processes, the queue() and shared signals should be in charge of the main process.
```

Comment:
This is never called. `platform.recover_platform_job_logs()`  has a call to `_exit` and catches any exceptions before calling that. Maybe we should remove that one inside `recover_platform_job_logs`, so that it has less responsibilities (i.e. it recovers only, without exiting the system).